janus_server/src/message.rs

```diff
@@ -158,9 +166,20 @@ fn authenticated_decode<M: Decode>(
 pub struct Duration(pub(crate) u64);
 
 impl Duration {
+    /// Create a duration representing the provided number of seconds.
     pub fn from_seconds(seconds: u64) -> Self {
         Self(seconds)
     }
+
+    /// Create a duration representing the provided number of minutes.
+    pub fn from_minutes(minutes: u64) -> Self {
+        Self::from_seconds(60 * minutes)
```

Comment:
Optional: since we're being pedantic -- do a checked multiply for `from_minutes` & `from_hours`?